samples/gen-go-blog/server/router.go

```diff
@@ -59,28 +61,40 @@ func (s *Server) Serve() error {
 	ingestURL := os.Getenv("TRACING_INGEST_URL")
 	isLocal := os.Getenv("_IS_LOCAL") == "true"
 	if (tracingToken != "" && ingestURL != "") || isLocal {
-		// Add rate limited sampling. We will only sample [Param] requests per second
-		// and [MaxOperations] different endpoints. Any endpoint above the [MaxOperations]
-		// limit will be probabilistically sampled.
-		cfgSampler := &jaegercfg.SamplerConfig{
-			Type:          jaeger.SamplerTypeRateLimiting,
-			Param:         5,
-			MaxOperations: 100,
+		samplingRate := .01               // 1% of requests
+		lowerBoundRateLimiter := 1.0 / 60 // 1 request/minute/operation
```

Comment:
on first look this seems really low. Is it 1 request/sec/operation?

Reply:
This was based on their docstring
https://godoc.org/github.com/uber/jaeger-client-go#GuaranteedThroughputProbabilisticSampler

Reply:
🤔 sounds like a good place to start I guess. hopefully 1% of traffic will mostly be more than this.